saleor/tests/e2e/orders/test_order_is_expired_after_specific_time.py

```diff
@@ -0,0 +1,115 @@
+import time
+
+import pytest
+
+from ....order.tasks import expire_orders_task
+from ..channel.utils import update_channel
+from ..checkout.utils import checkout_create, checkout_delivery_method_update
+from ..product.utils.preparing_product import prepare_product
+from ..shop.utils.preparing_shop import prepare_shop
+from ..utils import assign_permissions
+from .utils import order_create_from_checkout, order_query
+
+
+@pytest.mark.e2e
+def test_order_is_expired_after_specific_time_CORE_0214(
+    e2e_staff_api_client,
+    e2e_app_api_client,
+    permission_manage_products,
+    permission_manage_channels,
+    permission_manage_product_types_and_attributes,
+    permission_manage_shipping,
+    permission_manage_orders,
+    permission_manage_payments,
+    permission_manage_checkouts,
+    permission_handle_checkouts,
+):
+    # Before
+    permissions = [
+        permission_manage_products,
+        permission_manage_channels,
+        permission_manage_shipping,
+        permission_manage_product_types_and_attributes,
+        permission_manage_orders,
+        permission_manage_checkouts,
+        permission_handle_checkouts,
```

Comment:
I think those two are only needed for app so you can remove it from staff permission